k8s/ingress.yml

```diff
@@ -0,0 +1,23 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: mongo-ingress
+spec:
+  rules:
+  - http:
+      paths:
+      - path: /mongo-express
+        pathType: Prefix
+        backend:
+          service:
+            name: mongo-express-service
+            port:
+              number: 8081
+      - path: /mongo
+        pathType: Prefix
+        backend:
+          service:
+            name: mongodb-service
+            port:
+              number: 27017
+      

```

Comment:
Please remove this extra line at the end of file.

Reply:
done.